benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+file(GLOB BENCHMARK "*.cpp")
+add_executable( benchmark ${BENCHMARK} )
+
+target_link_libraries( benchmark fc Boost::program_options )
+target_include_directories( benchmark PUBLIC
+                            ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
This will need to be wrapped in quotes to handle spaces. You may also be able to just put a `.` here since paths are relative to `CMAKE_CURRENT_SOURCE_DIR` anyways. Or just forego it all together and `include "benchmark.hpp"` but that's typically not the pattern we use.

Reply:
Thanks for the suggestions.